### PR TITLE
Update porn.txt

### PR DESCRIPTION
**Pull request by lordyavin**
## Checklist

- [x] I have verified that I have not modified the following files:
      - inside the `adguard` folder
      - inside the `alt-version` folder
      - inside the `dnsmasq-version` folder
      - everything.txt

Because those files will be automatically updated using GitHub Actions!

### CODE REVIEW

File: porn.txt [hunk 1]
```diff
@@ -484624,6 +484628,8 @@
 0.0.0.0 xhamester.com
 0.0.0.0 xhamester.org
 0.0.0.0 xhammer.com
+0.0.0.0 xhammer.net
+0.0.0.0 xhammer.be
```
Comment (spirillen) [inline porn.txt:484632]: redirect to https://xhammer.azurewebsites.net/

<details><summary>Logger output</summary>

| | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/favicon.ico` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_back.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentMain.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_5.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_6.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_7.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_4.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_8.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_3.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_2.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/infob_1.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentBottom.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/website.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/community.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/events.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/battlenights.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/xhammer.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/contentTop.gif` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationLogon.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationForum.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationLocation.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationHome.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | image | `https://xhammer.azurewebsites.net/images/navigationStrip.jpg` |
| +2 |  |  | xhammer.azurewebsites.net | 1 | get | css | `https://xhammer.azurewebsites.net/css/xhammer.css` |
| +2 | *$document,permissions=browsing-topics=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=idle-detection=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=join-ad-interest-group=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=run-ad-auction=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | *$document,permissions=attribution-reporting=(),from=~localhost | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | no-remote-fonts: * true | -- | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +2 | no-remote-fonts: * true | -- | xhammer.azurewebsites.net | 1 | get | inline-font | `https://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.azurewebsites.net | 1 | get | doc | `https://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.azurewebsites.net | 1 | get | doc | `http://xhammer.azurewebsites.net/` |
| +1 |  |  | xhammer.be | 1 | get | image | `http://xhammer.be/favicon.ico` |
| +1 | *$document,permissions=browsing-topics=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=idle-detection=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=join-ad-interest-group=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=run-ad-auction=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | *$document,permissions=attribution-reporting=(),from=~localhost | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | no-remote-fonts: * true | -- | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +1 | no-remote-fonts: * true | -- | xhammer.be | 1 | get | inline-font | `http://xhammer.be/` |
| +1 |  |  | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
| +0 |  |  | xhammer.be | 1 | get | doc | `https://xhammer.be/` |
| +0 |  |  | xhammer.be | 1 | get | doc | `http://xhammer.be/` |
</details>

File: porn.txt [hunk 2]
```diff
@@ -346731,6 +346734,7 @@
 0.0.0.0 pornoroute.com
 0.0.0.0 pornorscorn.com
 0.0.0.0 pornorubka.net
+0.0.0.0 pornoruhe.net
```
Comment (spirillen) [inline porn.txt:346737]:
```
The page isn’t redirecting properly

An error occurred during a connection to www.pornoruhe.net.

    This problem can sometimes be caused by disabling or refusing to accept cookies.
```

File: porn.txt (hunk 1 above)
Comment (lordyavin) [reply porn.txt:484632]: So what?

File: porn.txt (hunk 2 above)
Comment (lordyavin) [reply porn.txt:346737]: @spirillen I have no idea what you want me to change. Please add

Comment (spirillen) [reply porn.txt:346737]: This is not necessary a request for change, it is a indication that I (me @spirillen ) is unable to confirm this is a porn domain, as the server do not redirect probable (miss configured) when I visited it. 

So this is more like a info + request for other to try confirm this as porn domain

**Comment by lordyavin** [comment]
👍

**Pull request merged**